portfolio/src/main/webapp/script.js

```diff
@@ -131,5 +148,10 @@ function initMap() {
 
 document.addEventListener("DOMContentLoaded", function() {
   renderForm();
-  initMap();
+  makeMap();
+  document.getElementById("number").addEventListener("input", function() {
+    if (validateForm())
+        getComments();
```

Comment:
The JavaScript style guide enforces [braces for all control structures](https://google.github.io/styleguide/jsguide.html#formatting-braces-all). This does meet the condition for putting it all on one line if you think that improves readability. I personally always use the braces though.

Reply:
Added braces. Think multi-line with braces is more readable/cleaner even though putting it on one line is more fun to write.